interface/patient_file/summary/demographics.php

```diff
@@ -161,14 +163,22 @@ function image_widget($doc_id, $doc_catg)
 <html>
 
 <head>
-<?php html_header_show();?>
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<?php //html_header_show();?>
+<!--<link rel="stylesheet" href="<?php /*echo $css_header;*/?>" type="text/css">
 <link rel="stylesheet" type="text/css" href="../../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
-<script type="text/javascript" src="../../../library/textformat.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="../../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-7-2/index.js"></script>
-<script type="text/javascript" src="../../../library/js/common.js?v=<?php echo $v_js_includes; ?>"></script>
-<script type="text/javascript" src="../../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
+<script type="text/javascript" src="../../../library/textformat.js?v=<?php /*echo $v_js_includes; */?>"></script>
+<script type="text/javascript" src="../../../library/dialog.js?v=<?php /*echo $v_js_includes; */?>"></script>
+
+<script type="text/javascript" src="<?php /*echo $GLOBALS['assets_static_relative']; */?>/jquery-min-3-1-1/index.js"></script>
+<script type="text/javascript" src="<?php /*echo $GLOBALS['assets_static_relative'] */?>/jquery-ui-1-12-1/jquery-ui.min.js"></script>
+<link rel="stylesheet" href="<?php /*echo $url . $GLOBALS['assets_static_relative']; */?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
+<script src="<?php /*echo $url . $GLOBALS['assets_static_relative']; */?>/bootstrap-3-3-4/dist/js/bootstrap.min.js" type="text/javascript"></script>
+<script type="text/javascript" src="../../../library/js/common.js?v=<?php /*echo $v_js_includes; */?>"></script>-->
+
+    <?php Header::setupHeader(['common', 'jquery-ui', 'jquery-ui-sunny']); ?>
+    <link rel="stylesheet" type="text/css" href="../../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
+    <script type="text/javascript" src="../../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
+
```

Comment:
To make this more clear, better to remove stuff completely than comment it out.

Reply:
Is a jquery-ui css (ie. sunny) really needed?

Reply:
How are you avoiding the fancybox/jquery 3.1.1 conflicts?

Reply:
Like to leave removals till cleanup so I know from whence I came :) The jquery cursor styles will be moved from tabs css to core css to cover both UI's. In this case, I fixed the jquery conficts. The main one is an easy fix which is the jquery load event. Also I've noted we have to get our dependencies in order because I see several cases where we will load jquery then turn around and reload, negating previous dependent dependencies on prior jquery load... A mouthful too be sure....